package.json

```diff
@@ -23,8 +23,8 @@
     "typecheck": "tsc && tsc -p integration_tests",
     "test": "jest --runInBand --detectOpenHandles",
     "security_audit": "npx audit-ci --config audit-ci.json",
-    "int-test": "cypress run --config video=false",
-    "int-test-ui": "cypress open",
+    "int-test": "cypress run --config video=false --env $(grep ONE_TIME_CODE_AUTH_ENABLED feature.env)",
```

Comment:
This is how the integration tests know about the value of the feature flag.
The feature flag is an env variable defined in `feature.env` as `ONE_TIME_CODE_AUTH_ENABLED`. We simply get it out of the file using grep and pass it into cypress on the command line.
Within the cypress runtime it is made available via `Cypress.env` (see `featureFlags.ts` above ☝️ )

Reply:
Is there a better way of doing this? with https://docs.cypress.io/guides/guides/environment-variables#Option-1-configuration-file maybe?

Reply:
Thats a fair challenge @ldlharper . I did look at that option when developing this, but felt we kinda already have this in a 'configuration file' in that its already in `feature.env` - I didnt really want to create another env file because the config option on both files would need to be kept in sync